modules/graceful-shutdown.adoc

```diff
@@ -1,102 +1,90 @@
 // Module included in the following assemblies:
 //
 // * backup_and_restore/graceful-cluster-shutdown.adoc
-
 :_content-type: PROCEDURE
 [id="graceful-shutdown_{context}"]
 = Shutting down the cluster
-
 You can shut down your cluster in a graceful manner so that it can be restarted at a later date.
-
 [NOTE]
 ====
 You can shut down a cluster until a year from the installation date and expect it to restart gracefully. After a year from the installation date, the cluster certificates expire.
 ====
-
 .Prerequisites
-
 * You have access to the cluster as a user with the `cluster-admin` role.
 * You have taken an etcd backup.
-
-.Procedure
-
-. If you plan to shut down the cluster for an extended period of time, determine the date that cluster certificates expire.
 +
-You must restart the cluster prior to the date that certificates expire. As the cluster restarts, the process might require you to manually approve the pending certificate signing requests (CSRs) to recover kubelet certificates.
-
-.. Check the expiration date for the `kube-apiserver-to-kubelet-signer` CA certificate:
+[IMPORTANT]
+====
+It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues when restarting the cluster.
+For example, the following conditions can cause the restarted cluster to malfunction:
+* etcd data corruption during shutdown
+* Node failure due to hardware
+* Network connectivity issues
+If your cluster fails to recover, follow the steps to restore to a previous cluster state.
+====
```

Comment:
the content in block "====" didn't have correct format. It showed like this:
It is important to take an etcd backup before performing this  procedure so that your cluster can be restored if you encounter any  issues when restarting the cluster. For example, the following conditions can cause the restarted cluster to  malfunction: * etcd data corruption during shutdown * Node failure due to hardware * Network connectivity issues If your cluster fails to recover, follow the steps to restore to a  previous cluster state.

And "restore to a previous cluster state." in the last sentence should be attached a link jumping relevant page.

Reply:
Removed because the same content was repeated in https://github.com/openshift/openshift-docs/pull/64574/files